bin/pygrb/pycbc_pygrb_plot_null_stats

```diff
@@ -46,33 +47,65 @@ __program__ = "pycbc_pygrb_plot_null_stats"
-    trigger/injection file"""
+# Function to load trigger data
+def load_data(input_file, vetoes, ifos, opts, injections=False):
+    """Load data from a trigger/injection file"""
 
-    # Inizialize the dictionary
-    data = {}
-    data['snr'] = None
     null_stat_type = opts.y_variable
+
+    # Initialize the dictionary
+    data = {}
+    data['coherent'] = None
     data[null_stat_type] = None
 
-    # Fill the dictionary in with required data
     if input_file:
         if injections:
-            trigs_or_injs = ppu.load_injections(input_file, vetoes, sim_table=sim_table)
+            logging.info("Loading injections...")
+            # This will eventually become ppu.load_injections()
+            trigs_or_injs = ppu.load_triggers(input_file, vetoes)
         else:
+            logging.info("Loading triggers...")
             trigs_or_injs = ppu.load_triggers(input_file, vetoes)
-        data['snr'] = numpy.asarray(trigs_or_injs.get_column('snr'))
-        if null_stat_type == 'coincident':
-            data[null_stat_type] = (trigs_or_injs.get_column('coinc_snr'))
-        elif null_stat_type == 'nullstat':
-            data[null_stat_type] = numpy.asarray(trigs_or_injs.get_column('null_statistic'))
-        elif null_stat_type == 'overwhitened':
-            data[null_stat_type] = numpy.asarray(trigs_or_injs.get_null_snr())
+            
+        num_trigs_or_injs = len(trigs_or_injs['network/end_time_gc'][:])
+
+        # Coherent SNR is always used
+        data['coherent'] = trigs_or_injs['network/%s_snr' % null_stat_type][:]
+
+        if null_stat_type == 'null':
+            data[null_stat_type] = trigs_or_injs['network/%s_snr' % null_stat_type][:]
+        elif null_stat_type == 'single':
+            att = ifo[0].lower()
+            data[null_stat_type] = trigs_or_injs['%s/snr_%s' % (opts.ifo, att)][:]
+        elif null_stat_type == 'coincident':
+            # Obtain all single SNRs
+            single_snr = {}
+            for ifo in ifos:
+                ifo_att = {'G1': 'g', 'H1': 'h1', 'H2': 'h2', 'L1': 'l',
+                           'V1': 'v', 'T1': 't'}
+                att = ifo_att[ifo]
+                single_snr[ifo] = trigs_or_injs['%s/snr_%s' % (ifo, att)][:]
+            
+            # Calculate coincident SNR
+            if len(ifos) > 1:
+                # Initialize some dictionaries we will use
+                single_snr_sq = dict((ifo, None) for ifo in ifos)
+                snr_sum_square = numpy.zeros(num_trigs_or_injs)
+                for ifo in ifos:
+                    # First square the individual SNR's
+                    single_snr_sq[ifo] = numpy.square(single_snr[ifo])
+                    # Then add them
+                    snr_sum_square = numpy.add(snr_sum_square,
+                                               single_snr_sq[ifo])
+                # Now obtain the square root
+                data[null_stat_type] = numpy.sqrt(snr_sum_square)
```

Comment:
It looks like if `null_stat_type == 'coincident'` and `len(ifos) == 1`, `data[null_stat_type]` does not get filled up.